<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HMIG7V7" failures="1" tests="4" name="PetTest" time="9.08" errors="0" timestamp="2025-03-17T14:45:51 IST">
  <testcase classname="api.testcase.UserTests2" name="testPostUser" time="3.075"/>
  <testcase classname="api.testcase.UserTests2" name="testGetUserByName" time="1.406"/>
  <testcase classname="api.testcase.UserTests2" name="testUpdateUserByName" time="2.724"/>
  <testcase classname="api.testcase.UserTests2" name="testDeleteUserByName" time="1.331">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.testcase.UserTests2.testDeleteUserByName(UserTests2.java:84)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- PetTest -->
